### PR TITLE
LV95 print stuff 

### PR DESCRIPTION
**Pull request by procrastinatio**
[Testlink with multiprint ](https://mf-geoadmin3.int.bgdi.ch/mom_lv95_print/index.html?print_url=//service-print.int.bgdi.ch)

- [x] Internal layer
- [x] Timeenabled layers (Zeitreihen, LUBIS)
- [x] Time series
- [x] External WMTS (e.g http://wmts20.geo.admin.ch/EPSG/2056/1.0.0/WMTSCapabilities.xml)
- [x] External WMS
- [x] Legend
- [x] Gitter
- [x] ToD

:bomb: Client reprojected layers won't be printed without any warning

merge and deploy https://github.com/geoadmin/service-print/pull/44 if you do not intent to use **print_url** parameter

`service-print`is ready: https://github.com/geoadmin/service-print/releases/tag/r171019

### CODE REVIEW

File: src/components/print/PrintLayerService.js [hunk 1]
```diff
@@ -426,8 +427,7 @@ goog.require('ga_urlutils_service');
         matrixSet: source.getMatrixSet() || '2056'
       });
 
-      if (!isExternalWmts) {
-
+      if (!isExternalWmts && gaGlobalOptions.defaultEpsg === 'EPSG:21781') {
```
Comment (oterral) [inline src/components/print/PrintLayerService.js]: defaultEpsg will never be 21781 anymore so we can probably remove all this if.

**Comment by procrastinatio** [comment]
jenkins build

**Comment by oterral** [comment]
jenkins test

**Comment by oterral** [comment]
jenkins test

**Comment by procrastinatio** [comment]
So Github close a PR if 0 commit and reopen it if at least one commit

**Pull request merged**